layouts/partials/button/_button.scss

```diff
@@ -82,7 +82,6 @@ $button-hamburger-hover-br: transparent;
 
 // Secondary button
 .button-secondary {
```

Comment:
A better approach with the buttons is to create a `.button` base class, containing all the common properties for buttons, and then create button variants like primary, secondary, etc… This will end up in your HTML with something like this:

```
<button class="button button-primary button-large">
  ...
</button>
```

In BEM:


```
<button class="button button--primary button--large">
  ...
</button>
```

Reply:
Interesting proposal. Thanks!